src/roomsync.rs

```diff
@@ -235,33 +235,36 @@ pub struct YRoomMessage {
     pub payloads: PyObject,
     #[pyo3(get)]
     pub broadcast_payloads: PyObject,
+    #[pyo3(get)]
+    pub has_edits: PyObject,
 }
 
 fn make_payloads(py: Python, payloads: &[Vec<u8>]) -> PyObject {
     PyList::new(py, payloads.iter().map(|payload| PyBytes::new(py, payload))).into()
 }
 
 impl YRoomMessage {
-    fn from_payloads(payloads: &[Vec<u8>], broadcast_payloads: &[Vec<u8>]) -> YRoomMessage {
+    fn from_payloads(payloads: &[Vec<u8>], broadcast_payloads: &[Vec<u8>], has_edits: bool) -> YRoomMessage {
         Python::with_gil(|py| YRoomMessage {
             payloads: make_payloads(py, payloads),
             broadcast_payloads: make_payloads(py, broadcast_payloads),
+            has_edits: PyBool::new(py, has_edits).into(),
```

Comment:
The pattern `has_edits.into_py(py)` is slightly nicer here and in the other cases `PyBool` is used.

Reply:
Thanks for the tip!